playbooks/roles/foreman_proxy_content/tasks/koji_repos.yml

```diff
@@ -0,0 +1,29 @@
+---
```

Comment:
I'd rather not get into the business of trying to refactor things alongside the rename, we could fix this after.